### ISSUE TITLE
What's the difference between images 1024x1024 and tfrecords on the FFHQ download page?

### ISSUE THREAD

**Issue by Delicious-Bitter-Melon** [EndlessSora/DeceiveD#3]
What's the difference between images 1024x1024 and tfrecords on the FFHQ download page?
The tfrecords have 273 GB, and the images 1024 x 1024 have 89.2 GB.
If I use Pytorch, can I directly download images 1024x1024 and use dataset_tool.py to create FFHQ 256x256?

**Comment by EndlessSora** [comment]
There is no significant difference. The tfrecords are just a data format the original authors used for their TensorFlow version training. They may also do certain data preprocessing when preparing this format. As for your dataset preparation, please refer to the [Dataset Preparation](https://github.com/EndlessSora/DeceiveD#dataset-preparation) section in this repo. You may find more useful tips in  [stylegan2-ada-pytorch Preparing datasets](https://github.com/NVlabs/stylegan2-ada-pytorch#preparing-datasets).

**Comment by Delicious-Bitter-Melon** [comment]
Thank you. How do you set the value of snap when training on FFHQ-1K. Do you use the default 50 in your paper? I found that small datasets are easy to over-fitting, and 50 is not a reasonable value.

**Comment by woctezuma** [comment]
I don't think `--snap` is an important parameter for training. It is a matter of how frequent snapshots are saved to disk.

> The training exports network pickles (`network-snapshot-<INT>.pkl`) and example images (`fakes<INT>.png`) at regular intervals (controlled by `--snap`). For each pickle, it also evaluates FID (controlled by `--metrics`) and logs the resulting scores in `metric-fid50k_full.jsonl` (as well as TFEvents if TensorBoard is installed).

**Comment by EndlessSora** [comment]
You can make it more frequent by setting a smaller `--snap` value if you need to carefully find a more reasonable result. But it would also increase the evaluation time during training. Please set `--aug=apa --with-dataaug=true` to apply APA together with compatible standard data augmentations to obtain possible relatively reasonable results. You might also try a smaller `--target` value on small datasets. More details can be found in our paper.
